File: Shruti/WFTextEditor/BaseBilling.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WFTextEditor
{
    public abstract class BaseBilling : IBilling
    {
        public virtual double CalculateBill(double cost, double tax)
        {
            return cost + (cost * tax);
        }

        public double CalculateCost(double quantity, double rate)
        {
            return quantity * rate;
        }
    }
}

File: NotePad - Malathi/NotePadApp/app/src/main/java/com/trimble/notepadapp/billing/BaseBilling.java
package com.trimble.notepadapp.billing;

/**
 * Created by mbommi on 10/11/2018.
 */

public class BaseBilling implements IBilling {
    @Override
    public double calculateCost(int quantity, double rate) {
        return quantity * rate;
    }

    @Override
    public double calculateBill(double cost, double tax) {
        return cost + cost * tax;
    }
}
